<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetJog</class>
 <widget class="QWidget" name="widgetJog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>243</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="2">
      <widget class="StyledToolButton" name="cmdXPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>X+</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/images/1401562173_chevron-right.png</normaloff>:/images/1401562173_chevron-right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="adjustSize" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="StyledToolButton" name="cmdXMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>X-</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/images/1401561986_chevron-left.png</normaloff>:/images/1401561986_chevron-left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="adjustSize" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="StyledToolButton" name="cmdYMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Y-</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/images/1401562173_chevron-down.png</normaloff>:/images/1401562173_chevron-down.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="adjustSize" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="StyledToolButton" name="cmdZMinus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Z-</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../images.qrc">
           <normaloff>:/images/1401562699_icon-arrow-down-b.png</normaloff>:/images/1401562699_icon-arrow-down-b.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="adjustSize" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="StyledToolButton" name="cmdStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/images/brake.png</normaloff>:/images/brake.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="adjustSize" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="StyledToolButton" name="cmdZPlus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Z+</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../images.qrc">
           <normaloff>:/images/1401562699_icon-arrow-up-b.png</normaloff>:/images/1401562699_icon-arrow-up-b.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="adjustSize" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="StyledToolButton" name="cmdYPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Y+</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/images/1401562173_chevron-up.png</normaloff>:/images/1401562173_chevron-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="adjustSize" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Step:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ComboBoxKey" name="cboJogFeed">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Feed:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBoxKey" name="cboJogStep">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <item>
        <property name="text">
         <string>Continuously</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkKeyboardControl">
     <property name="text">
      <string>Keyboard control</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledToolButton</class>
   <extends>QToolButton</extends>
   <header>styledtoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxKey</class>
   <extends>QComboBox</extends>
   <header>widgets/comboboxkey.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdYPlus</sender>
   <signal>pressed()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdYPlusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdYPlus</sender>
   <signal>released()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdYPlusReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdZPlus</sender>
   <signal>pressed()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdZPlusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdZPlus</sender>
   <signal>released()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdZPlusReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdZMinus</sender>
   <signal>pressed()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdZMinusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdZMinus</sender>
   <signal>released()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdZMinusReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdXMinus</sender>
   <signal>pressed()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdXMinusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdXMinus</sender>
   <signal>released()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdXMinusReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdXPlus</sender>
   <signal>pressed()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdXPlusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdXPlus</sender>
   <signal>released()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdXPlusReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdYMinus</sender>
   <signal>pressed()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdYMinusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdYMinus</sender>
   <signal>released()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdYMinusReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKeyboardControl</sender>
   <signal>toggled(bool)</signal>
   <receiver>widgetJog</receiver>
   <slot>onChkKeyboardControlToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdStop</sender>
   <signal>clicked()</signal>
   <receiver>widgetJog</receiver>
   <slot>onCmdStopClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCmdYPlusPressed()</slot>
  <slot>onCmdYPlusReleased()</slot>
  <slot>onCmdYMinusPressed()</slot>
  <slot>onCmdYMinusReleased()</slot>
  <slot>onCmdXPlusPressed()</slot>
  <slot>onCmdXPlusReleased()</slot>
  <slot>onCmdXMinusPressed()</slot>
  <slot>onCmdXMinusReleased()</slot>
  <slot>onCmdZPlusPressed()</slot>
  <slot>onCmdZPlusReleased()</slot>
  <slot>onCmdZMinusPressed()</slot>
  <slot>onCmdZMinusReleased()</slot>
  <slot>onChkKeyboardControlToggled(bool)</slot>
  <slot>onCmdStopClicked()</slot>
 </slots>
</ui>
